### ISSUE TITLE
Sidebar: save folds / unfold by default;

### ISSUE THREAD

**Issue by irvnriir** [tim-we/tabs-aside#84]
The sidebar should save folding state of the Sessions, and restore it every time its shown . Or they may should be unfold at start .

**Comment by tim-we** [comment]
I agree that the folding state should be remembered. My current plan is to use IndexedDB for storing additional data such as this, favicons, thumbnails, etc. 

It is possible to have multiple sidebar instances (multiple windows), what do you think should happen in this scenario? Should the folding state be synchronized across all sidebars? If not which state should be stored?

**Comment by irvnriir** [comment]
I'm not sure how multiple sidebars is related here, its just about that having the Sessions items unfolded in sidebar is usually more ergonomic than folded .

**Comment by irvnriir** [comment]
With multiple browser windows . If the window represent a Session, then save for this session, otherwise from the last window that stayed open .

**Comment by irvnriir** [comment]
With multiple browser windows . Restore for reopening (at least Sessions), and not sync between windows .
